usr/share/rear/backup/BLOCKCLONE/default/500_start_clone.sh

```diff
@@ -32,3 +42,10 @@ case "$(basename ${BLOCKCLONE_PROG})" in
 esac
 
 StopIfError "Failed to create archive with $BLOCKCLONE_SOURCE_DEV"
+
+# If $BLOCKCLONE_SOURCE_DEV was initially mounted AND successfully unmounted, 
+# try to remount it before leaving
+if [ "$umount_res" = "0" ]; then
+    LogPrint "Trying to remount $mp"
+    mount $v $mp
```

Comment:
Yes, I was aware of the fact that it would only work for mountpoints defined in `/etc/fstab`. I took the view that anything not present there would be transient anyway and that silently failing to remount it was acceptable behaviour -- if far from ideal.
But the approach you suggest is better by far, so I'll try to implement that ;-)